### PR TITLE
added aggregation of count sites

### PR DESCRIPTION
**Pull request by juanfonsecaLS1**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Will take a look now...

File: README.Rmd [hunk 1]
```diff
@@ -259,18 +259,24 @@ ADF_dtype |>
 library(sf)
 library(tmap)
 ```
-The following code reads the network with the estimated commute trips from the npt repository
+The following code reads the network with the estimated commute trips from
+the npt repository. Each edge/link of the network has four attributes. We will
+focus on `bicycle` which is the estimated number of daily commute trips in both
```
Comment (Robinlovelace) [inline README.Rmd]: Good plan to focus on current (well 2011!) cycling estimates.

File: README.Rmd [hunk 2]
```diff
@@ -259,18 +259,24 @@ ADF_dtype |>
...
+directions.
 
 ```{r}
 rnet_commute = read_rds("../npt/outputs/rnet_commute.Rds")
 rnet_commute
 ```
-A `sf` object is created from the `clean_counts` data frame.
+A `sf` object is created from the `clean_counts` data frame. AADF for each
```
Comment (Robinlovelace) [inline README.Rmd]:
```suggestion
An `sf` object is created from the `clean_counts` data frame. AADF for each
```

File: README.Rmd [hunk 3]
```diff
@@ -283,19 +289,15 @@ A subset of the counts are taken based on a buffer of the `rnet_commute` object.
...
 sf_counts_selected = sf_counts[rnet_buffer20,]
-
 ```
 
 ### Approach A
 The nearest feature is joined to each point location
 ```{r}
-sf_counts_joined = st_join(sf_counts_selected,rnet_commute,join = st_nearest_feature)
-sf_counts_joined
+val_app1 = st_join(sf_counts_selected,rnet_commute,join = st_nearest_feature)
```
Comment (Robinlovelace) [inline README.Rmd:299]: Logical approach.

File: README.Rmd [hunk 4]
```diff
@@ -323,6 +325,107 @@ val_app1 |>
...
+tmap_mode("view")
+tm_shape(sel_counts_buf30[sel_counts_buf30$siteID %in% c("EDH0040","EDH0041"),])+
+  tm_polygons(alpha = 0.5)+
+  tm_shape(sf_counts_selected)+
+  tm_dots()
+  
+```
+
+
+A more complex instance is the overlap of sensors on Melville Dr, two of the
```
Comment (Robinlovelace) [inline README.Rmd]: Did this improve the counter-network fit?

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
Great job Juan, feel free to merge, I've made a minor `suggestion` and have a few comments but overall :+1:

File: README.Rmd (hunk 4 above)
Comment (juanfonsecaLS1) [reply README.Rmd]: There is a small increase in the R<sup>2</sup>, although the 2011 estimates are very low (~0.7% of the counts). I have a added a linear model at the end of each approach to show this in be7440f. I am planning to include a simple relative difference and GEH comparison.

**Comment by juanfonsecaLS1** [comment]
Hi Robin, I think I don't think I have write access to merge the branch

**Review by Robinlovelace — APPROVED** [review]
Looking great, let's merge this to keep the momentum going!

**Pull request merged**